### PR TITLE
Allow kpartx and disk compression with kiwi

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: src/bci_build/package/kiwi.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
         package_list=[
             "btrfsprogs",
             "checkmedia",
+            "dosfstools",
```
Comment (dcermak) [inline src/bci_build/package/kiwi.py:31]:
```suggestion
```

File: src/bci_build/package/kiwi.py [hunk 2]
```diff
@@ -36,16 +37,19 @@
...
             "kiwi-systemdeps-core",
+            "kpartx",
             "libxml2-devel",
             "lvm2",
             "make",
             "netcat-openbsd",
             "python3-devel",
             "python3-kiwi",
             "python3-pip",
+            "squashfs",
```
Comment (dcermak) [inline src/bci_build/package/kiwi.py:48]:
```suggestion
            "kiwi-systemds-disk-images",
            "libxml2-devel",
            "lvm2",
            "make",
            "netcat-openbsd",
            "python3-devel",
            "python3-kiwi",
            "python3-pip",
```

**Review by dcermak — COMMENTED** [review]
We can rely on kiwi's metapackages to cut the package list a bit

File: src/bci_build/package/kiwi.py (hunk 2 above)
Comment (dirkmueller) [reply src/bci_build/package/kiwi.py:48]: that doesn't exist here. we can use kiwi-systemdeps-filesystems though hopefully

**Pull request merged**